bot/exts/evergreen/githubinfo.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import random
+from datetime import datetime
+from typing import Optional
+
+import aiohttp
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from bot.constants import NEGATIVE_REPLIES
+
+log = logging.getLogger(__name__)
+
+
+class GithubInfo(commands.Cog):
+    """Fetches info from GitHub."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @staticmethod
+    async def fetch_data(url: str) -> dict:
+        """Retrieve data as a dictionary."""
+        async with aiohttp.ClientSession() as cs:
+            async with cs.get(url) as r:
+                return await r.json()
+
+    @commands.command(name='github', aliases=['gh'])
+    @commands.cooldown(1, 5, BucketType.user)
+    async def get_github_info(self, ctx: commands.Context, username: Optional[str]) -> None:
+        """
+        Fetches a user's GitHub information.
+
+        Username is optional and sends the help command if not specified.
+
+        Usage: .gh [username]
+        """
+        if username is None:
+            await ctx.invoke(self.bot.get_command('help'), 'github')
+            return
```

Comment:
We should reset the command cooldown if no username is given.
```suggestion
        if username is None:
            await ctx.invoke(self.bot.get_command('help'), 'github')
            ctx.command.reset_cooldown(ctx)
            return
```